### PR TITLE
 fix(collapse): [collapse] fix manually change the modelValue of a Collapse component, the change event does not trigger

### PR DESCRIPTION
**Pull request by wkif**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-vue/blob/dev/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: examples/sites/demos/pc/app/collapse/events.vue [hunk 1]
```diff
@@ -18,6 +18,7 @@
       <div>结果可控：用户可以自由的进行操作，包括撤销、回退和终止当前操作等。</div>
     </tiny-collapse-item>
   </tiny-collapse>
+  <button @click="click">点击触发</button>
```
Comment (wuyiping0628) [inline examples/sites/demos/pc/app/collapse/events.vue]: 1.demo里面要使用tiny-button  2.events-composition-api.vue要同步修改

**Review by wuyiping0628 — CHANGES_REQUESTED** [review]

File: examples/sites/demos/pc/app/collapse/events.vue (hunk 1 above)
Comment (wkif) [reply examples/sites/demos/pc/app/collapse/events.vue]: 了解

File: packages/vue/src/collapse/src/index.ts
Comment (AcWrong02) [inline packages/vue/src/collapse/src/index.ts:1]: 这个change事件应该不用加在这里，before-close是要传给组件的prop没错，但是change是组件要emit的事件

File: packages/renderless/src/collapse/vue.ts
Comment (AcWrong02) [inline packages/renderless/src/collapse/vue.ts:1]: 考虑得全面一点的话，其实当activeNames.value.push('2')的时候，change事件也是不会触发的

Comment (AcWrong02) [reply packages/renderless/src/collapse/vue.ts:1]: 我看element也有这个问题。。。感觉这种场景几乎不会出现

Comment (AcWrong02) [reply packages/renderless/src/collapse/vue.ts:1]: 另外，这样子当你点击collapse-item的时候是会触发两次change事件的

**Review by AcWrong02 — CHANGES_REQUESTED** [review]

File: packages/renderless/src/collapse/vue.ts
Comment (wkif) [reply packages/renderless/src/collapse/vue.ts:1]: 貌似没有好的解决思路了

Comment (AcWrong02) [reply packages/renderless/src/collapse/vue.ts:1]: 导致触发两次change事件的原因是：你点击了collapse-item的时候会触发update:modeValue和change事件，触发了update:modeValue的时候，接下去就会触发watch里面的change事件。
可以尝试添加一个变量，判断是不是点击了collapse-item触发的change，如果是的话，就不触发watch中的change事件

**Comment by kagol** [comment]
触发两次change事件的问题未解决，并且冲突长时间未解决

**Pull request closed without merging**